packages/core/src/lib/wallet/wallet.types.ts

```diff
@@ -113,6 +113,8 @@ export interface BrowserWalletSignAndSendTransactionsParams
 export type BrowserWalletBehaviour = Modify<
   BaseWalletBehaviour,
   {
+    buildImportAccountsUrl?(): string;
```

Comment:
arguments to this interface has been removed from this commit. Is there a reason why?

Reply:
Yes, it no longer needs parameters:

>  `buildImportAccountsUrl()` (browser wallets) returns the URL for the import accounts page on the wallet. The encrypted JSON string will be appended onto it as a fragment.

The idea is they return the URL for importing accounts in the current network, and we append the encrypted data after a `#`. Now the wallet implementation never needs encrypted data and we've enforced that encrypted data is added as a fragment.

I believe we discussed this last week but I don't think I updated the drafted proposal.

Reply:
I assumed that what you are saying is that calling `buildImportAccountsUrl()` will return `https://www.example-wallet.io&someparams#` and we append encrypted data next after `#`?

But would it enforce the url expression to always have `#` at the end of the url. Would it be more developer friendly if we just take encrypted string as argument of a function and return completed url?

```
(encryptedData) => `https://www.example-wallet.io#${encryptedData}&someparams`
```